src/cli_plugin/lib/error_if_x_pack.js

```diff
@@ -17,15 +17,15 @@
  * under the License.
  */
 
-import { isOSS } from './is_oss';
+import { isOss } from './is_oss';
 
 function isXPack(plugin) {
   return /x-pack/.test(plugin);
 }
 
 export function errorIfXPackInstall(settings) {
```

Comment:
Do we even need any of this logic anymore? X-Pack cannot be installed or removed from a distributable

Reply:
Yeah, that's what this logic is intended for. If someone tries to install x-pack this logic will help people know they should download the distributable.